l10n_it_fatturapa_out/tests/data/IT06363391001_00009.xml

```diff
@@ -0,0 +1,104 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Hai controllato il file su https://sdi.fatturapa.gov.it/SdI2FatturaPAWeb/AccediAlServizioAction.do?pagina=controlla_fattura ?

Reply:
Sì!